eng/pipelines/runtime.yml

```diff
@@ -51,1248 +51,1234 @@ pr:
-    - windows_x86
-    - Linux_musl_x64
-    helixQueuesTemplate: /eng/pipelines/coreclr/templates/helix-queues-setup.yml
-    helixQueueGroup: libraries
-    jobParameters:
-      testScope: innerloop
-      liveRuntimeBuildConfig: checked
-      condition: >-
-        or(
-          eq(dependencies.evaluate_paths.outputs['SetPathVars_coreclr.containsChange'], true),
-          eq(variables['isRollingBuild'], true))
-
-#
-# Release Libraries Test Execution against a checked runtime
-# Only if CoreCLR or Libraries is changed
-#
-- template: /eng/pipelines/common/platform-matrix.yml
-  parameters:
-    jobTemplate: /eng/pipelines/libraries/run-test-job.yml
-    buildConfig: Release
-    platforms:
-    - Linux_musl_arm
-    - Linux_musl_arm64
-    - Linux_x64
-    - windows_x64
-    helixQueuesTemplate: /eng/pipelines/coreclr/templates/helix-queues-setup.yml
-    helixQueueGroup: libraries
-    jobParameters:
-      testScope: innerloop
-      liveRuntimeBuildConfig: checked
-      condition: >-
-        or(
-          eq(dependencies.evaluate_paths.outputs['SetPathVars_coreclr.containsChange'], true),
-          eq(variables['isRollingBuild'], true))
-
-- template: /eng/pipelines/common/platform-matrix.yml
+extends:
+  template:  /eng/pipelines/common/templates/single-stage-pipeline-with-resources.yml
   parameters:
-    jobTemplate: /eng/pipelines/libraries/run-test-job.yml
-    buildConfig: ${{ variables.debugOnPrReleaseOnRolling }}
-    platforms:
-    - OSX_x64
-    helixQueuesTemplate: /eng/pipelines/coreclr/templates/helix-queues-setup.yml
-    helixQueueGroup: libraries
-    jobParameters:
-      testScope: innerloop
-      liveRuntimeBuildConfig: checked
-      condition: >-
-        or(
-          eq(dependencies.evaluate_paths.outputs['SetPathVars_coreclr.containsChange'], true),
-          eq(dependencies.evaluate_paths.outputs['SetPathVars_libraries.containsChange'], true),
-          eq(variables['isRollingBuild'], true))
+    jobs:
+
+    #
+    # Evaluate paths
+    #
+    - ${{ if eq(variables.dependOnEvaluatePaths, true) }}:
+      - template: /eng/pipelines/common/evaluate-default-paths.yml
+
+    #
+    # Build CoreCLR checked
+    # Only when CoreCLR is changed
+    #
+    - template: /eng/pipelines/common/platform-matrix.yml
+      parameters:
+        jobTemplate: /eng/pipelines/coreclr/templates/build-job.yml
+        buildConfig: checked
+        platforms:
+        - Linux_x86
+        - Linux_x64
+        - Linux_arm
+        - Linux_arm64
+        - Linux_musl_arm
+        - Linux_musl_arm64
+        - Linux_musl_x64
+        - OSX_arm64
+        - Tizen_armel
+        - windows_x86
+        - windows_x64
+        - windows_arm
+        - windows_arm64
+        jobParameters:
+          testGroup: innerloop
+          condition: >-
+            or(
+              eq(dependencies.evaluate_paths.outputs['SetPathVars_coreclr.containsChange'], true),
+              eq(variables['nonWasmRuntimeTestsContainsChange'], true),
```

Comment:
could you please double check the changes in this file? It looks like these are wrong merges.